api-publishing/src/main/java/org/zalando/nakadi/EventPublishingController.java

```diff
@@ -133,6 +131,10 @@ private ResponseEntity postEventInternal(final String eventTypeName,
             reportMetrics(eventTypeMetrics, result, totalSizeBytes, eventCount);
             reportSLOs(startingNanos, totalSizeBytes, eventCount, result, eventTypeName, client);
 
+            if (result.getStatus() == EventPublishingStatus.FAILED) {
```

Comment:
Was it because of custom 207 processing in TracingFilter?

Reply:
Yes, this should result in marking `207` responses with the `error` boolean flag.